lighthouse-core/test/lib/i18n/i18n-test.js

```diff
@@ -265,5 +283,13 @@ describe('i18n', () => {
         // eslint-disable-next-line max-len
         .toThrow(`Provided value "sirNotAppearingInThisString" does not match any placeholder in ICU message "Hello {timeInMs, number, seconds} World"`);
     });
+
+    it('formats correctly with NaN and Infinity numeric values', () => {
+      const helloInfinityStr = str_(UIStrings.helloBytesWorld, {in: Infinity});
+      expect(helloInfinityStr).toBeDisplayString('Hello ∞ World');
```

Comment:
> `expect(helloInfinityStr).toBeDisplayString('Hello ∞ World');`

I guess this is how it's supposed to get formatted? 🤷‍♀

Reply:
◊◊◊

Reply:
∞∞∞∞∞∞∞v∞